src/components/Code/Code.js

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import sharedComponent from '../../sharedComponent';
 
 import Block from './CodeBlock';
 import Container from './CodeContainer';
 
 const CodeImpl = styled.pre.withConfig({ displayName: 'Code' })`
-  font-family: ${({ theme }) => theme.inlineCode.fontFamily};
-  fontSize: ${({ theme }) => theme.inlineCode.fontSize};
-  background: ${({ theme }) => theme.inlineCode.bg};
-  color: ${({ theme }) => theme.inlineCode.fg};
-  border: ${({ theme }) => theme.inlineCode.border};
-  padding: ${({ theme }) => theme.inlineCode.padding}
-  borderRadius: ${({ theme }) => theme.inlineCode.borderRadius};
+  font-family: ${({ theme }) => theme.fonts.monospace};
+  fontSize: 0.85em;
```

Comment:
FYI: `em` units are relative to their parent, just want to make sure that is the intention for this components font-size. 

Also, this should be `font-size` and not `fontSize`